src/go/rdctl/cmd/reset.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright © 2025 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/spf13/cobra"
+
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/client"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/config"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/factoryreset"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/paths"
+	"github.com/rancher-sandbox/rancher-desktop/src/go/rdctl/pkg/shutdown"
+)
+
+var (
+	vmReset      bool
+	k8sReset     bool
+	cacheReset   bool
+	factoryReset bool
+)
+
+var resetCmd = &cobra.Command{
+	Use:   "reset",
+	Short: "Reset Rancher Desktop",
+	Long: `Some reset options are combinations of others:
```

Comment:
Needs an extra line to describe the command (possibly repeating the short description):

```suggestion
	Long: `Delete parts of Rancher Desktop settings.
Some reset options are combinations of others:
```